features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/.eslintignore

```diff
@@ -4,3 +4,4 @@
 # Pleae comment the reason if you are  ignoring a file/directory here
 /source/src/app/components/Apis/Details/Security/**
 /source/src/app/components/Apis/Details/Documents/**
+/source/src/app/components/**
```

Comment:
Why did we expand the scope of ignored paths ?

```
/source/src/app/components/**
```
This will exclude/ignore everything that resides in the components directory ??